openml-python-common/src/main/java/com/feedzai/openml/python/ClassificationPythonModel.java

```diff
@@ -191,7 +192,11 @@ public int classify(final Instance instance) {
             asNotNullable = this.classToIndexConverter.apply(classValue);
         } catch (final NullPointerException e) {
```

Comment:
I've spent sometime now debugging this, and I cannot see how this catch clause is reached, since the function here underneath is a `Map#get`, which worst case scenario will return a `null`, which will be directly redirected to the variable. I can't also find any sort of documentation mentioning a `NullPointerException`. What am I missing?

**Note:** that I need to create a test that reaches this clause to be able to increase test coverage.
**Also Note:** that there is no clear injection point in this clause, so mocking won't work either.

Reply:
Since this assigning to an "int", if the function returns null, that will cause a null pointer exception, right?

Reply:
Completely forgot about that! You're right ;)